src/status_im/contexts/wallet/add_account/create_account/view.cljs

```diff
@@ -255,11 +256,12 @@
                                               {:on-complete
                                                #(rf/dispatch
                                                  [:keycard/connect-derive-address-and-add-account
-                                                  {:pin                  %
+                                                  {:pin %
                                                    :derived-from-address derived-from
-                                                   :key-uid              key-uid
-                                                   :derivation-path      @derivation-path
-                                                   :account-preferences  preferences}])}])))
+                                                   :key-uid key-uid
+                                                   :derivation-path (create-account.utils/normalize-path
+                                                                     @derivation-path)
```

Comment:
One other place within this same file/screen might need normalization: 

https://github.com/status-im/status-mobile/blob/ce1d2c4ed8d70147fc7ef4e938ec642fa3de5d15/src/status_im/contexts/wallet/add_account/create_account/view.cljs#L294

Reply:
good point